src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -151,14 +151,19 @@ const Amounts: React.FC<AmountsProps> = ({ sellToken, order }) => {
 }
 
 const Expires: React.FC<Pick<Props, 'order' | 'pending' | 'isPendingOrder'>> = ({ order, isPendingOrder }) => {
-  const { isNeverExpires, expiresOn } = useMemo(() => {
+  const { isNeverExpires, expiresOn, expireDateLocal } = useMemo(() => {
```

Comment:
nit: maybe `expireDateFormatted` instead of local?

Reply:
ok